tests/bindings/javascript/printer.test.js

```diff
@@ -1,4 +1,5 @@
 /*
+
```

Comment:
Why that empty line?

Reply:
[8b91978](https://github.com/cellml/libcellml/pull/987/commits/8b9197839bfd7444bcb1759aac1632098da1660f)